app/ui/base/src/main/java/com/fsck/k9/ui/base/ThemeManager.kt

```diff
@@ -44,11 +50,17 @@ class ThemeManager(
     val translucentDialogThemeResourceId: Int = themeProvider.translucentDialogThemeResourceId
 
     fun init() {
-        updateAppTheme()
+        generalSettingsManager.getSettingsFlow()
+            .map { it.appTheme }
+            .distinctUntilChanged()
+            .onEach {
+                updateAppTheme(it)
+            }
+            .launchIn(GlobalScope)
```

Comment:
Instead of `GlobalScope` please use "AppCoroutineScope". See e.g. 

https://github.com/k9mail/k-9/blob/09ea6bd7e8dd5badf5eda8dbeb18ffffda15ef66/app/ui/legacy/src/main/java/com/fsck/k9/ui/changelog/ChangeLogManager.kt#L19

https://github.com/k9mail/k-9/blob/09ea6bd7e8dd5badf5eda8dbeb18ffffda15ef66/app/ui/legacy/src/main/java/com/fsck/k9/ui/changelog/KoinModule.kt#L8